src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+    - content: Satoshi Nakamoto
+      key: D
+      order: 3
+- uuid: 3110ead9-fde0-43c0-8007-b1ba3a6781fd
+  type: SingleChoice
+  content: Like every digital asset Bitcoin has the problem of “double spend”?
+  hint: Double Spend means copying and reselling the same digital asset many number of times.
+  explanation: Every bitcoin has a unique hash pointer attached to it hence it becomes impossible to create a copy with a hash present in the system.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 552462b9-be2a-4d45-b3fd-665c86c799bc
+  type: MultipleChoice
+  content: What are the benefits of Bitcoin over fiat currency?
+  hint: Benefits of blockchain
+  explanation: Bitcoin has an immutable ledger , allows users ownership and is portable to carry.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Immutable ledger
+      key: A
+      order: 0
+    - content: User sovereignty
+      key: B
+      order: 1
+    - content: High tangible value
+      key: C
+      order: 2
+    - content: Portability
+      key: D
+      order: 3
+- uuid: 99349427-2e10-4649-9962-6b26fac55f05
+  type: MultipleChoice
+  content: What are the benefits of Fiat currency over Bitcoin?
+  hint: Benefits of centralised system
+  explanation: Fiat currency has low transaction fees , they are backed by gold and thus possess a high tangible value.
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Very less transaction Fess
+      key: A
+      order: 0
+    - content: Extremely volatile nature
+      key: B
+      order: 1
+    - content: Strong backing asset
+      key: C
+      order: 2
+    - content: High tangible value
+      key: D
+      order: 3
+- uuid: c12ae1b5-3940-496a-88fa-41be29142f44
+  type: SingleChoice
+  content: Where do you keep your cryptocurrency private key?
+  hint: NoHint
+  explanation: Cryptocurrency private key must be kept away from the internet.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Wallet
+      key: A
+      order: 0
+    - content: Bank Account 
+      key: B
+      order: 1
+    - content: Floppy disk
+      key: C
+      order: 2
+    - content: Drive Storages
+      key: D
+      order: 3
+- uuid: 948746c7-237e-4e73-bc50-3b714ecb1ccf
+  type: SingleChoice
+  content: Bitcoin mining works on which of the principle?
+  hint: NoHint
+  explanation: Proof of Work consensus mechanism is used to mine Bitcoin.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Proof of Authority
+      key: A
+      order: 0
```

Comment:
we can remove all `order: ` as they are not needed

Reply:
Done